### PR TITLE
Fix Static Analysis Issues for Sensor Module

### PR DESCRIPTION
**Pull request by vilasrk**

### CODE REVIEW

File: sensors/2.0/iiohal_mediation_v2.0/iioClient.cpp [hunk 1]
```diff
@@ -67,12 +67,16 @@ void iioClient::iioThread(struct iioclient_device *devlist) {
      * once before use
      */
     iioClient *iioc = iioClient::get_iioClient();
+	
+    if(iioc == NULL)
```
Comment (JaikrishnaNemallapudi) [inline sensors/2.0/iiohal_mediation_v2.0/iioClient.cpp:71]: space

File: sensors/2.0/iiohal_mediation_v2.0/libiio_client/iiod-client.c [hunk 2]
```diff
@@ -266,6 +268,8 @@ int iiod_client_get_trigger(struct iiod_client *client, void *desc,
     for (i = 0; i < nb_devices; i++) {
         struct iio_device *cur = iio_context_get_device(ctx, i);
 
+        if(cur == NULL)
```
Comment (JaikrishnaNemallapudi) [inline sensors/2.0/iiohal_mediation_v2.0/libiio_client/iiod-client.c:271]: space

**Review by JaikrishnaNemallapudi — APPROVED** [review]
looks good

**Review by JaikrishnaNemallapudi — APPROVED** [review]
lgtm

**Review by JeevakaPrabu — APPROVED** [review]

**Pull request merged**